.github/workflows/cherryPick.yml

```diff
@@ -104,17 +104,17 @@ jobs:
 
           \`\`\`bash
           git fetch
-          git checkout roryabraham-cherry-pick-staging-56382-1
-          git cherry-pick -S -x --mainline 1 6486b1d1961d4c6de85154905d57af5118b33ff9
+          git checkout ${{ github.actor }}-cherry-pick-staging-${{ github.event.inputs.PULL_REQUEST_NUMBER }}-${{ github.run_attempt }}
+          git cherry-pick -S -x --mainline 1 ${{ steps.getCPMergeCommit.outputs.MERGE_COMMIT_SHA }}
           \`\`\`
 
           Then manually resolve conflicts, and commit the change with \`git cherry-pick --continue\`. Lastly, please run:
 
           \`\`\`bash
-          git commit --amend -m "\$(git log -1 --pretty=%B)" -m "(CP triggered by roryabraham)"
+          git commit --amend -m "\$(git log -1 --pretty=%B)" -m "(CP triggered by ${{ github.actor }})"
           \`\`\`
 
-          That will help us keep track of who triggered this CP. Once all that's done, push your changes with \`git push origin roryabraham-cherry-pick-staging-6486b1d1961d4c6de85154905d57af5118b33ff9-1\`, and then open this PR for review.
```

Comment:
@roryabraham was this wrong the whole time or do we really need to use the commit hash here? It wouldn't make sense, since the branch name uses the PR number instead.

Reply:
was wrong the whole time